dashboard/src/components/ColorSchemeToggle.tsx

```diff
@@ -49,8 +49,8 @@ export const ColorSchemeToggle = (props: IconButtonProps) => {
       }}
       sx={[
         mode === colorMode.DARK
-          ? { "& > *:first-child": { display: "none" } }
-          : { "& > *:first-child": { display: "initial" } },
+          ? { "& > *:first-of-type": { display: "none" } }
+          : { "& > *:first-of-type": { display: "initial" } },
         mode === colorMode.LIGHT
           ? { "& > *:last-child": { display: "none" } }
           : { "& > *:last-child": { display: "initial" } },
```

Comment:
This is kinda hacky? why not move it to its own css file and do the checks there?

Reply:
converted to styled component